backend/dal/Migrations/20200722193504_Initial.cs

```diff
@@ -1,9 +1,11 @@
 ﻿using System;
 using Pims.Dal.Helpers.Migrations;
 using Microsoft.EntityFrameworkCore.Migrations;
+using System.Diagnostics.CodeAnalysis;
 
 namespace Pims.Dal.Migrations
 {
+    [ExcludeFromCodeCoverage]
```

Comment:
oh wow, of course this would get included, I suppose we'll get notified of missing coverage when we push PR's so hopefully we'll remember to do this in the future.

Reply:
I've updated the script to auto include them for us `make db-add n=v01.11.00`